tests/test_json_schema.py

```diff
@@ -2613,23 +2612,6 @@ class Model(TypedDict):
     }
 
 
-def test_typeddict_with_extra_behavior_allow():
-    class Model:
-        @classmethod
-        def __get_pydantic_core_schema__(cls, source_type: Any, handler: GetCoreSchemaHandler) -> CoreSchema:
-            return core_schema.typed_dict_schema(
-                {'a': core_schema.typed_dict_field(core_schema.str_schema())},
-                extra_behavior='allow',
-            )
-
-    assert TypeAdapter(Model).json_schema() == {
-        'type': 'object',
-        'properties': {'a': {'title': 'A', 'type': 'string'}},
-        'required': ['a'],
-        'additionalProperties': True,
-    }
```

Comment:
So, I think we should only be pulling the `extra_behavior` from the config here. If folks disagree, I can pull from the schema as well, though not sure why we have this setting there - seems redundant.

Reply:
extra items are going to be supported soon for typed dicts in Python. Is Pydantic support for it going to be easy to implement with these changes?